loading-button-android/src/main/java/br/com/simplepass/loadingbutton/presentation/ProgressButtonPresenter.kt

```diff
@@ -111,6 +111,7 @@ internal class ProgressButtonPresenter(private val view: ProgressButton) {
     fun doneLoadingAnimation(fillColor: Int, bitmap: Bitmap) {
         view.doneFillColor = fillColor
         view.doneImage = bitmap
+        view.initRevealAnimation()
```

Comment:
Since we are not keeping the same `fillColor` and `doneImage` initRevealAnimation should receive then as a parameter to the function, instead of using `view.doneImage`and `view.doneFillColor` and those can be completely removed from the `ProgressButton` interface. 

Like this:

```
override fun initRevealAnimation(fillColor: Int, bitmap: Bitmap) {
    revealAnimatedDrawable = createRevealAnimatedDrawable(fillColor, bitmap)
}
```
and
```
internal fun ProgressButton.createRevealAnimatedDrawable(fillColor: Int, bitmap: Bitmap): CircularRevealAnimatedDrawable =
    CircularRevealAnimatedDrawable(this, fillColor, bitmap).apply {
        val padding = Rect()
        drawableBackground.getPadding(padding)
        val paddingSides = (Math.abs(padding.top - padding.left))
        setBounds(paddingSides, padding.top, finalWidth - paddingSides, finalHeight - padding.bottom)
   callback = this@createRevealAnimatedDrawable
}
```